### PR TITLE
Profile Created

### PR DESCRIPTION
**Pull request by kshitij9896**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4662"></a>

### CODE REVIEW

**Review by stevenovak123 — APPROVED** [review]

File: data/kshitij9896.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      "icon": "FaTwitter",
+      "color": "#00ACEE"
+    },
+    {
+      "name": "Let's Connect On LinkedIn",
+      "url": "https://linkedin.com/in/kshitij-bansod",
+      "icon": "FaLinkedin"
+    }
+  ],
+
```
Comment (stevenovak123) [inline data/kshitij9896.json:36]: add `}` a the end

**Review by stevenovak123 — CHANGES_REQUESTED** [review]

File: data/kshitij9896.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      "url": "https://twitter.com/kshitij_bansod",
+      "icon": "FaTwitter",
+      "color": "#00ACEE"
+    },
+    {
+      "name": "Let's Connect On LinkedIn",
+      "url": "https://linkedin.com/in/kshitij-bansod",
+      "icon": "FaLinkedin"
+    }
+  ],
```
Comment (kumarsonsoff3) [inline data/kshitij9896.json:35]: There is an extra comma `,` which should be removed. 
```suggestion
  ]
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Unfortunately you've removed the closing bracket from the end. Please add a `}` in the end as Steven suggested.

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good to me. 🎉 

But not sure why did you delete your other sections & links from the file. When you ever wanted to add them again, you can follow [these docs for editing](https://linkfree.eddiehub.io/docs/how-to-guides/editing)

**Pull request merged**